src/content/configuration/watch.md

```diff
@@ -70,3 +70,44 @@ poll: 1000 // Check for changes every second
 
 T> If watching does not work for you, try out this option. Watching does not work with NFS and machines in VirtualBox.
 
+# Troubleshooting
+
+## Webpack doesn't recompile on change while watching
```

Comment:
I would rather see more concise titles here as very long ones won't be very intuitive or easy to display in our side nav, e.g.:

![image](https://user-images.githubusercontent.com/8988697/35315255-6074d2ca-0099-11e8-8743-98885723ad90.png)

For example, maybe the following would work better and still convey the idea:

``` md
### Missed Compilations
```

Please consider the same change for the rest of the longer headers. Also note that I changed this to an `h3` to properly nest it under:

``` md
## Troubleshooting
```

Another option would be to simply change them from headers to bold text, for example:

> ## Troubleshooting
> 
> If you are experiencing any issues, please see the following notes. There are a variety of reasons why
> webpack might miss a file change.
>
> __Changes are seen, but no files are updated__
> 
> ...
>
> __Not enough watchers__
>
> ...